orange-datamodel/src/main/java/com/hartwig/hmftools/datamodel/linx/LinxFusion.java

```diff
@@ -7,11 +7,32 @@
 
 @Gson.TypeAdapters
 @Value.Immutable
-@Value.Style(allParameters = true, passAnnotations = { NotNull.class, Nullable.class })
-public interface LinxFusion {
+@Value.Style(passAnnotations = { NotNull.class, Nullable.class })
+public interface LinxFusion
+{
+    @NotNull
+    String geneStart();
+
+    @NotNull
+    String geneContextStart();
+
+    @NotNull
+    String geneTranscriptStart();
+
+    @NotNull
+    String geneEnd();
 
     @NotNull
-    String name();
+    String geneContextEnd();
+
+    @NotNull
+    String geneTranscriptEnd();
+
+    @NotNull
+    default String name()
+    {
+        return String.format("%s-%s", geneStart(), geneEnd());
+    }
```

Comment:
I'd rename to `display` since this is effectively what it is. There should be no logic dependent on the actual implementation of this function and we have several other `display` functions in other places of the datamodels for similar purpose (mostly enums)